GNUmakefile

```diff
@@ -0,0 +1,63 @@
+TEST?=$$(go list ./... |grep -v 'vendor')
+GOFMT_FILES?=$$(find . -name '*.go' |grep -v vendor)
+WEBSITE_REPO=github.com/hashicorp/terraform-website
```

Comment:
Can you remove the bits in the Makefile that are building the terraform website, since that's not needed here?